### ISSUE TITLE
The meaning of "magnification" in the virtual atlas

### ISSUE THREAD

**Issue by giuseb** [giuseb/histologyguide#9]
I have been assuming that all virtual slides were acquired using either a 40x or a 60x objective, and that's what "magnification" in the Microscope Slide Information panel really means. Correct?

If so, experienced observers implicitly understand that the "40x" label indicates the magnifying power of the objective used for the digital scan, which has little to do with the actual magnification of the image on a computer screen.

On the other hand, I suspect that the term "magnification" may be confusing for many first-year medical students.

I would consider rewording, e.g. OBJECTIVE (or acquisition objective) instead of MAGNIFICATION in the Microscope Slide Information panel.

**Comment by giuseb** [comment]
I have also been assuming (again, please correct me if I am wrong) that slides were scanned only once, i.e. no 4x, 10x, or 20x objectives were actually used. If so, the actual meaning of the toolbar's "magnification" buttons is:
- the last one (usually 40x, in some cases 60x) sets the view at 100%, i.e. one image pixel per one screen pixel
- The other buttons zoom out proportionally by interpolating pixels, i.e. 4x = 10%, 10x = 25%, 20x = 50% of the above

During practice sessions, I often have to remind students that even though it says 40x, the actual magnification of the image on your screen is unrelated. On my 16" retina display, for example, it's more like 1000x, since the 50 µm scale bar is close to 50 mm wide.

Those 4-10-20-40 magic numbers do make sense as relative quantities to an experienced microscopist with an intuitive sense of how much magnification changes when switching objectives.

**Comment by ghost** [comment]
The maximum magnification shown in the toolbar (usually 40x with some 60x) is the magnification of the objective lens used to obtain the image.  To clarify, MAGNIFICATION will be changed to OBJECTIVE in the Microscope Slide Information Panel.
The slides were only scanned at the maximum objective magnification.  The relationship between the objective lens and digital zooming works as you described.
When viewing any digital photo, the inability to distinguish between physical lens magnification, digital magnification, display size, and pixel density is a big problem.  This an intuitive skill we learn because we know the actual physical size of things being viewed (e.g., cat versus mountain).  However, this must be learned again for small objects (microscopy) or large objects (astronomy).
Magnification/zoom is usually displayed relative to 100% for one pixel in a photo matching one pixel of the display. We chose to display zoom relative to an objective lens to help students learn/judge the sizes of objects on a digital display.  This made sense because students were also using physical microscopes in laboratories.  Unfortunately, this is lost when students only use textbooks, atlases, or websites to view slides.

**Comment by Brelje** [comment]
I changed "MAGNIFICATION" in the slide information to "OBJECTIVE."  This value is the magnification of the objective used by the slide scanner to obtain this image.  (However, most of the older Aperio scanners did not use a high NA 40x objective but rather scanned at a higher resolution with the 20x objective.  This is why the raw, scanned images appear much "softer" than seen by eye.  To address this, we have sharpened the images over time using different filters on at least two occasions.

The inability to determine the size of objects (or the related magnification) in any image is a problem.  For photographs taken with a cellphone, viewers usually have experience looking at the objects shown.  However, as you noted, students typically have little to no experience looking at tissues or cells through a physical microscope.  The solution used in research publications is either scale bars (most forms of light microscopy) or magnification of the final print size (electron microscopy, EM).  We do the former, but do not do the latter because of large differences in display sizes.  The magnification buttons in the toolbar are an attempt to provide some context, but, as you noted, this fails for most students.  The total size of the view on the right side of the toolbar is probably the best indicator of size.  I suspect most students do not make this connection.

Examining other websites, they typically do not even display a scale bar.  I have examined viewers for large light or electron microscopy datasets and maps, but have not found any better methods.

One approach I tried but rejected was to place a subtle grid across the image. However, like debris or artifacts, this becomes the first thing you "see" in the image.

**Comment by giuseb** [comment]
All things considered, I would favor the following:

- the scale bar is an absolute must, let's try to catch any remaining obvious mismatches
- remove the 4x 10x 20x 40x buttons; I have never seen a student use them anyway
- state the actual objective used for the original scan; this piece of information is quite relevant, it would be nice to have it shown at all times, for example, in place of the above "magnification" buttons.

**Comment by Brelje** [comment]
You make good points.  The magnification buttons were an attempt to provide some connection to the scale seen in the microscope.  At most, this is useful only if you have previous experience with a physical microscope (which almost all students do not).  So, removing the magnification buttons makes sense.
 
Potential Replacements:
- "1.6x | 40x Objective" (1.6x is replace with current value and 40x by the correct one - 40x or either 60x)
- "1.6x | 250 x 100 um" (current magnification with the total view size - currently shown on the right side of the toolbar).  I'm not sure if this is obvious, but I would have to use the tooltip to explain.

**Comment by Brelje** [comment]
The 2nd option looks too "busy."

The shorter "Lens" appears more suitable than "Objective" and is easier to comprehend.

[MH 001 with no mag buttons](http://160.94.203.145/slideview/MH-001-nuclear-morphology/01-slide-1.html)

**Comment by Brelje** [comment]
I also sent this to you via email, so Bob would receive this response.

Bob and Giuseppe,
Giuseppe questioned the utility of the magnification buttons (i.e., 4x, 10x, 20x, 40x) on the left side of the toolbar.  Even to him, the correlation between these values and the image being shown was confusing.  This made me reconsider this feature.
Problems with Current Magnification and Magnification Buttons:
* Buttons labeled 4x, 10x, 20x, 40x, mimic microscope objectives – not the displayed image.  I am guessing that many people think they show different images.
* Students do not think in terms of microscope objectives unless they’ve had extensive lab experience.  This is unlikely for our visitors.
* This UI is predicated on changing the magnification before finding a region of interest.  This is the opposite of the usual approach.  Typically, you identify a region and then repeatedly click on it to zoom in.
* Displayed magnifications refer to scanner optics, not how the image is currently being shown.  How do you distinguish one from the other?
Giuseppe has commented that his students do not use them.  After reconsidering them, this makes sense given these problems.
Proposed Solution:
http://160.94.203.145/slideview/MH-001-nuclear-morphology/01-slide-1.html
* Replace the objective buttons with a label showing the lens used to acquire the image (e.g., “40x Lens” or “60x Lens”).
* Magnification is replaced by the image zoom shown as a percentage.  This aligns with user expectations based on other software (e.g., photos app on your cell phone, image processing apps, browser, etc.).  It is muvh easier to understand that “100%” refers to the full resolution of the image..
* Hover the mouse over the current zoom value, and a pop-up menu with common presets (5%, 10%, 25%, etc.) is shown.  This mimics standard UX patterns.  Additionally, the lowest zoom value is always a 'Fit'  option, like the “Home” button on the toolbar.
* Hover the mouse over the “40x Lens” and the tooltip shows the size of the image in pixels and actual units
This UI is more likely to be closer to how our users already know and digital image norms.
Let me know what you think,
Todd Clark Brelje

**Comment by giuseb** [comment]
I find the proposed solution excellent. Thanks!
Let me know if I can close this issue.
